.travis.yml

```diff
@@ -0,0 +1,76 @@
+sudo: required
+services:
+  - docker
+
+language: python
+
+stages:
+  - test
+  - delivery
+  - submit
+
+jobs:
+  include:
+    - stage: test
+      python: 2.7
+      env: TOXENV=py27-codeclimate
+      install:
+          - pip install tox
+      script:
+          - tox
+
+    - stage: test
+      python: 3.6
+      env: TOXENV=py36-codeclimate
+      before_script:
+          - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+          - chmod +x ./cc-test-reporter
+          - ./cc-test-reporter before-build
+      install:
+          - pip install tox
+      script:
+          - tox
+      after_script:
+          - mv tests/coverage.xml .
+          - ./cc-test-reporter after-build -t coverage.py --exit-code $TRAVIS_TEST_RESULT tests/coverage.xml
+
+    - stage: delivery
+      if: type != pull_request
+      env:
+        - FOLDER=/package
+        - PACKAGE_NAME=python-hanadb_exporter
```

Comment:
@arbulu89 As this is a application by itself, we don't need to call the package with the "python" prefix. It is only for python modules. I would call the package hanadb_exporter only.

Reply:
@diegoakechi Ok, I will rename the package to `hanadb_exporter`